package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "next lint --dir src",
     "format": "prettier --write src",
     "prepare": "husky install",
-    "schema": "npx get-graphql-schema http://localhost:8000/graphql/ > schema.graphql",
+    "schema": "npx -y get-graphql-schema http://localhost:8000/graphql/ > schema.graphql",
```

Comment:
Don't hesitate to group your changes in multiple commits

Reply:
Okay